appcues/src/main/java/com/appcues/Appcues.kt

```diff
@@ -270,6 +271,8 @@ public class Appcues internal constructor(internal val scope: AppcuesScope) {
      */
     public fun setPushToken(token: String?) {
         storage.deviceToken = token
+
+        analyticsTracker.track(AnalyticsEvent.DeviceUpdated.eventName, properties = null, interactive = false, isInternal = true)
```

Comment:
checked and confirmed - this will only track if the session exists (user identified), that check is inside `AnalyticsTracker`

Reply:
I think that makes sense and its aligned with what matt is doing on iOS